portal/models/intervention_strategies.py

```diff
@@ -182,7 +182,7 @@ def update_user_card_html(intervention, user):
             """.format(user.display_name)
             link_label = 'Begin questionnaire'
             link_url = url_for('assessment_engine_api.present_assessment',
-                               instrument_id='epic26')
+                               instrument_ids='epic26')
```

Comment:
The main `present-assessment` API should now use querystring parameters exclusively.

I moved the old route to a new, deprecated function that can be easily removed once we get everyone on the new API/route.